jipdate.py

```diff
@@ -227,7 +232,7 @@ def parse_status_file(jira, filename):
     # [SWG-28]
     # [LITE-32]
     # etc ...
-    regex = r"^\[([A-Z]+-\d+)\]\n$"
+    regex = r"^\[(.*)\]\n$"
```

Comment:
I'm fine with using the patch you have provided for this and then we can later see if there is another way of doing this. I don't think we need to do all "by-the-book" in the script, especially not know when it is in a very early stage.